### PR TITLE
Slider: optimize store subscriber calls

### PR DESCRIPTION
**Pull request by fbasso**
Just to avoid to call `storeName$()` multiple time in computed.

### CODE REVIEW

File: core/lib/slider.ts [hunk 1]
```diff
@@ -297,13 +297,13 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 	const sliderDomRectOffset$ = computed(() => (vertical$() ? sliderDomRect$().top : sliderDomRect$().left));
 	const sliderDomRectSize$ = computed(() => (vertical$() ? sliderDomRect$().height : sliderDomRect$().width));
 	const sortedValues$ = computed(() => [...values$()].sort((a, b) => a - b));
-	const sortedHandles$ = computed(() =>
-		values$()
+	const sortedHandles$ = computed(() => {
+		return values$()
```
Comment (divdavem) [inline core/lib/slider.ts:301]: Here I prefer the previous syntax as it is a single expression.

Comment (fbasso) [reply core/lib/slider.ts:301]: Oh yes, it's just because it's going to be change with the next PR I'm working on, on top of this one...

**Comment by fbasso** [comment]
Thanks for your comments, @divdavem and @quentinderoubaix, I pushed back again with the changes...

**Review by quentinderoubaix — APPROVED** [review]
LGTM!

**Comment by fbasso** [comment]
Thanks ;)

**Pull request merged**